.gitignore

```diff
@@ -28,6 +28,9 @@ packages/**/*.jar
 *.jar
 local-docs/_site
 
+# openapi generator
+!**/5.0.0.jar
```

Comment:
This is here because of the jetty mock server, right? Do we still have this mock server? If not we should remove the gitignore entries.